xarray/tests/test_interp.py

```diff
@@ -573,3 +573,16 @@ def test_cftime_to_non_cftime_error():
 
     with pytest.raises(TypeError):
         da.interp(time=0.5)
+
+
+@requires_cftime
+@requires_scipy
+def test_cftime_interp():
```

Comment:
Do you think you can find a better name for this test?  It does not seem to use cftime.